src/features/DisplayControls/DrugChart/styles/DrugChart.scss

```diff
@@ -51,6 +53,27 @@
   scrollbar-width: none; /* Firefox */
 }
 
+.arrow-button {
```

Comment:
The class name is too generic, It might bite us back in future. You might have to wrap it in the drug-chart class

Reply:
Thanks, I've moved it